ETT-Backend/Controllers/EmployeeController.cs

```diff
@@ -17,10 +17,11 @@ public EmployeeController(IEmployeeService employeeService)
             _EmployeeService = employeeService;
         }
 
-        [HttpGet("{email}")]
+        [HttpGet]
         [Authorize]
-        public ActionResult<EmployeeResponse> Get(string email)
+        public ActionResult<EmployeeResponse> Get()
         {
+            var email = User.FindFirst(x => x.Type == "email").Value;
```

Comment:
Where does the User come from? Is it the process that calls Get( ) ?

Reply:
It comes from the Identity of the user who called the method.  Which comes from the Jwt